nupic/research/frameworks/dendrites/modules/dendritic_mlp.py

```diff
@@ -67,83 +72,101 @@ class DendriticMLP(nn.Module):
                     input
     """
 
-    def __init__(self, input_size, output_size, hidden_size, num_segments, dim_context,
-                 kw=True, dendrite_sparsity=0.95, weight_init="modified",
-                 dendrite_init="modified", freeze_dendrites=False,
-                 dendritic_layer_class=AbsoluteMaxGatingDendriticLayer):
+    def __init__(
+        self, input_size, output_size, hidden_sizes, num_segments, dim_context,
+        kw, kw_percent_on=0.05, context_percent_on=1.0, weight_sparsity=0.95,
+        weight_init="modified", dendrite_init="modified", freeze_dendrites=False,
+        dendritic_layer_class=AbsoluteMaxGatingDendriticLayer, output_nonlinearity=None
+    ):
```

Comment:
minor comment: I'd recommend putting `output_nonlinearity` before `dendritic_layer_class` since the former is likely to be specified by the user more frequently.